doc/interface/unit_operations/radial_flow_models.rst

```diff
@@ -26,7 +26,7 @@ For information on model equations, refer to :ref:`lumped_rate_model_without_por
 
 ``COL_DISPERSION``
 
-   �Radial dispersion coefficient
+   �Radial dispersion coefficient
```

Comment:
There appears to be some formatting issue.

Reply:
![image](https://github.com/modsim/CADET/assets/86779218/b5fde664-60e9-4146-8507-128f48239af0)

What the character in front of Radial is displayed as in Github changed like 3 times and now settled on this 😂. I will just delete it.

Reply:
Oh look 
![image](https://github.com/modsim/CADET/assets/86779218/7fce93ed-291d-4f6f-8aa9-3085121adf3e)